docs/docs/how-tos/debug-nebari.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: debug-nebari
+title: Debug Nebari deployment
+description: Debug your Nebari deployment
+---
+
+## Debug the Kubernetes Cluster
+
+If you need more information about your Kubernetes cluster, [`k9s`](https://k9scli.io/) is a terminal-based UI that is extremely useful for debugging. It simplifies navigating, observing, and managing your applications in Kubernetes. `k9s` continuously monitors Kubernetes clusters for changes and provides shortcut commands to interact with the observed resources. It's a fast way to review and resolve day-to-day issues in Kubernetes, a huge improvement to the general workflow, and a best-to-have tool for debugging your Kubernetes cluster sessions.
+
+[Installation instructions for macOS, Windows, and Linux](https://github.com/derailed/k9s) are available.
+
+By default, `k9s` starts with the standard directory that's set as the context (in this case Minikube). To view all the current process press 0:
+
+![Image of the k9s terminal UI](/img/k9s_UI.png)
+
+> **NOTE**: In some circumstances, you will be confronted with the need to inspect any services launched by your cluster at your `localhost`. For instance, if your cluster has problem with the network traffic tunnel configuration, it may limit or block the user’s access to destination resources over the connection.
```

Comment:
This note is very difficult to understand, especially the first sentence.
 - Why would I need to "inspect any service"? Is there a specific service I need to inspect?
 - Why is `localhost` involved? Is that where I'm launching the services? Or where my cluster is running? Or where I'm viewing the cluster from?
 - "confronted with the need" sounds adversarial; perhaps "in some cases you may need to"

In the second sentence, the object of "it" after the comma is unclear: is "it" referring to your cluster, or the network config?

Also in the second sentence, there's a syntax error: "if your cluster has problem**s**" or "if your network has **a** problem".

Reply:
agreed. this needs a bunch more information before it will make sense to me.